protocol/xmrmaker/event.go

```diff
@@ -11,8 +11,9 @@ import (
 // EventType ...
 type EventType byte
 
+// ...
```

Comment:
I'd like to get good descriptions on both the maker and taker event constants before we merge. All events should clearly state if they are triggered by the counterparty, contract logs, or something in our own code.